nixos/modules/services/x11/urserver.nix

```diff
@@ -0,0 +1,72 @@
+# urserver service
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.urserver;
+in {
+
+  options.services.urserver = {
+
+    enable = mkEnableOption "urserver";
+
+    openFirewall = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Open ports in the firewall to interact with the urserver.
+      '';
+    };
+
+    usedTCPPorts = mkOption {
+      type = types.listOf types.port;
+      default = [ 9510 9512 ];
+      description = ''
+        TCP ports that are used by urserver.
+        By default port 9510 is used for the web server.
+        By default port 9512 is used for WiFi connections.
```

Comment:
It might be sufficient to deal with the firewall reminder, with an additional line here saying that ports should be opened in order for the service to be exposed.

Reply:
Looking at https://github.com/Infinisil/rfcs/blob/config-option/rfcs/0042-config-option.md#default-values and the example below it seems desirable that the default firewall ports are opened when the service is enabled?
I'm pushing a very minimal service implementation now. The proposed `settings` bit is still just a proposal, right? It didn't seem to work for me.